img/s/swile.co.svg

```diff
@@ -0,0 +1 @@
+<svg width="48" height="48" xmlns="http://www.w3.org/2000/svg"><defs><linearGradient x1="-2.661%" y1="0%" x2="100%" y2="100%" id="a"><stop stop-color="#CE55B0" offset="0%"/><stop stop-color="#F34DAC" offset="17.128%"/><stop stop-color="#F48E66" offset="31.654%"/><stop stop-color="#F0B832" offset="43.449%"/><stop stop-color="#B2CE37" offset="59.06%"/><stop stop-color="#63D7AF" offset="75.078%"/><stop stop-color="#1CC2DF" offset="88.597%"/><stop stop-color="#0580E6" offset="100%"/></linearGradient></defs><g fill="none" fill-rule="evenodd"><path d="M35.693.96c4.008 0 5.474.383 6.943 1.169a7.776 7.776 0 013.235 3.235c.786 1.469 1.169 2.935 1.169 6.943h0v23.386c0 4.008-.383 5.474-1.169 6.943a7.776 7.776 0 01-3.235 3.235c-1.469.786-2.935 1.169-6.943 1.169h0-23.386c-4.008 0-5.474-.383-6.943-1.169a7.776 7.776 0 01-3.235-3.235C1.343 41.167.96 39.701.96 35.693h0V12.307c0-4.008.383-5.474 1.169-6.943a7.776 7.776 0 013.235-3.235C6.833 1.343 8.299.96 12.307.96h0z" stroke="url(#a)" stroke-width="1.92" fill="#000"/><path d="M35.834 15.073c-3.541 1.73-3.896 6.029-4.182 9.482-.326 3.639-.624 4.635-1.388 4.635-.906 0-1.125-2.762-1.26-5.352-.208-3.78-.494-8.958-5.136-8.958-4.618 0-4.903 5.178-5.111 8.96-.135 2.588-.354 5.35-1.26 5.35-1.542 0-1.542-7.806-1.542-14.079v-.128H12v.128c0 11.23 0 18.009 5.497 18.009 4.72 0 5.006-5.237 5.214-9.066.078-1.702.24-5.244 1.157-5.244.885 0 1.058 3.07 1.173 5.102l.01.147c.208 3.824.493 9.061 5.213 9.061 4.636 0 5.03-4.76 5.317-8.236l.004-.046c.227-2.696.46-5.485 1.977-6.194l.118-.055-1.73-3.573-.116.057z" fill="#FFF"/></g></svg>
```

Comment:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 48 48"><defs><linearGradient id="a" x1="-2.7%" x2="100%" y1="0%" y2="100%"><stop offset="0%" stop-color="#CE55B0"/><stop offset="17.1%" stop-color="#F34DAC"/><stop offset="31.7%" stop-color="#F48E66"/><stop offset="43.4%" stop-color="#F0B832"/><stop offset="59.1%" stop-color="#B2CE37"/><stop offset="75.1%" stop-color="#63D7AF"/><stop offset="88.6%" stop-color="#1CC2DF"/><stop offset="100%" stop-color="#0580E6"/></linearGradient></defs><g fill="none" fill-rule="evenodd"><path fill="#000" stroke="url(#a)" stroke-width="1.9" d="M35.693.96c4.008 0 5.474.383 6.943 1.169a7.776 7.776 0 0 1 3.235 3.235c.786 1.469 1.169 2.935 1.169 6.943v23.386c0 4.008-.383 5.474-1.169 6.943a7.776 7.776 0 0 1-3.235 3.235c-1.469.786-2.935 1.169-6.943 1.169H12.307c-4.008 0-5.474-.383-6.943-1.169a7.776 7.776 0 0 1-3.235-3.235C1.343 41.167.96 39.701.96 35.693V12.307c0-4.008.383-5.474 1.169-6.943a7.776 7.776 0 0 1 3.235-3.235C6.833 1.343 8.299.96 12.307.96z"/><path fill="#FFF" d="M35.834 15.073c-3.541 1.73-3.896 6.029-4.182 9.482-.326 3.639-.624 4.635-1.388 4.635-.906 0-1.125-2.762-1.26-5.352-.208-3.78-.494-8.958-5.136-8.958-4.618 0-4.903 5.178-5.111 8.96-.135 2.588-.354 5.35-1.26 5.35-1.542 0-1.542-7.806-1.542-14.079v-.128H12v.128c0 11.23 0 18.009 5.497 18.009 4.72 0 5.006-5.237 5.214-9.066.078-1.702.24-5.244 1.157-5.244.885 0 1.058 3.07 1.173 5.102l.01.147c.208 3.824.493 9.061 5.213 9.061 4.636 0 5.03-4.76 5.317-8.236l.004-.046c.227-2.696.46-5.485 1.977-6.194l.118-.055-1.73-3.573-.116.057z"/></g></svg>
```

Here's an SVG that uses `viewBox` instead of `width` and `height`